file_link.rb

```diff
@@ -0,0 +1,15 @@
+
+module File_link
+  
+  def self.write(name,desig,dept)
+    file_name = "employeedetails.csv"
```

Comment:
what if I ask you to also store the detail of a department in a csv file? Instead of hardcoding the file name, take it as a parameter